src/components/extensive/WizardForm/FormHeader.tsx

```diff
@@ -17,28 +17,30 @@ export const WizardFormHeader = (props: WizardFormHeaderProps) => {
   const t = useT();
 
   return (
-    <div className="border-primary-50 sticky top-[74px] z-40 flex bg-leavesWithOverlay bg-cover bg-center p-6">
-      <div className="mx-auto flex w-full min-w-0 max-w-7xl items-center gap-8">
+    <div className="sticky top-[74px] z-40 flex border-neutral-150 bg-treesHeaderWithOverlay bg-cover bg-no-repeat px-10 py-3 md:py-8">
+      <div className="mx-auto flex w-full min-w-0 max-w-[82vw] items-center gap-8">
         <div className="min-w-0 flex-grow text-white">
           {props.title && (
-            <Text variant="text-bold-subtitle-600" className="overflow-hidden text-ellipsis whitespace-nowrap">
+            <Text variant="text-36-bold" className="overflow-hidden text-ellipsis whitespace-nowrap">
               {props.title}
             </Text>
           )}
-          <Text variant="text-bold-body-300" className="overflow-hidden text-ellipsis whitespace-nowrap">
-            <T _str="Progress: {number} steps complete" number={`${props.currentStep}/${props.numberOfSteps}`} />
-          </Text>
-          <Text variant="text-bold-caption-200">
-            <Switch>
-              <Case condition={props.formStatus === undefined}>{t("Unsaved")}</Case>
-              <Case condition={props.formStatus === "saving"}>{t("Saving…")}</Case>
-              <Case condition={props.formStatus === "saved"}>{t("Saved")}</Case>
-              <Case condition={props.errorMessage}>{t("Something went wrong.")}</Case>
-            </Switch>
-          </Text>
+          <div className="flex">
+            <Text variant="text-14-light" className="overflow-hidden text-ellipsis whitespace-nowrap">
+              <T _str="Progress: {number} steps complete" number={`${props.currentStep}/${props.numberOfSteps}`} />
+            </Text>
+            <Text variant="text-14-light">
+              <Switch>
+                <Case condition={props.formStatus === undefined}>{t("Unsaved")}</Case>
```

Comment:
I think this condition will be better use as `condition={!props.formStatus}`

Reply:
done